### ISSUE TITLE
-bash: ./sinusbot: cannot execute binary file: Exec format error

### ISSUE THREAD

**Issue by HiveSolution** [flyth/ts3soundbot#442]
I setup everything as instructed by the quide, but running ./sinusbot i get this error.

`-bash: ./sinusbot: cannot execute binary file: Exec format error`

I'm running a Ubuntu 14.04.3 LTS on a Samsung Exynos5422 with 2GB RAM (oDroid XU4 Unit)

Never encountered that problem before, and couldn't find anything.

Could the problem be the ARM architecture?

**Comment by LoloLaurentiu** [comment]
chmod 755 sinusbot ?

**Comment by HiveSolution** [comment]
Yes. Already done. Also tried running as root with parameter and just to test with chmod 777.

Still the same error message

**Comment by maxibanki** [comment]
You have a 32 bit System right? The Sinusbot requires a 64 Bit operating system.
